clients/python/girder_client.py

```diff
@@ -482,7 +485,11 @@ def downloadItem(self, itemId, dest, name=None):
                     break
                 else:
                     dest = os.path.join(dest, self._transformFilename(name))
-                    os.makedirs(dest)
+                    try:
+                        os.makedirs(dest)
+                    except:
```

Comment:
Can you make this `except OSError as exception: if exception.errno != errno.EEXIST:` so we don't catch other exceptions? (see girder/__init__.py)